recipes/geoh5-interop/recipe.yaml

```diff
@@ -0,0 +1,66 @@
+context:
+  name: geoh5_interop
+  version: 1.1.1
+  python_min: 3.10
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: b5d42898e812da1420e348309048a8bad1d56cdc11c8ca00a4828d3ac3fefae0
+
+build:
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry-core >=1.0.0
+    - python ${{ python_min }}.*
+    - setuptools
+  run:
+    - python >=${{ python_min }}
+    - geoh5py >=0.10.0,<0.11.dev
+    - las-geoh5 >=0.3.0,<0.4.dev
+    - mira-omf >=3.3.0,<3.4.dev
+
+tests:
+  - python:
+      imports:
+      - geoh5_interop
+      # dependency imports
+      - geoh5py
+      - las_geoh5
+      - omf
+      - properties
+      pip_check: false  # something wrong with properties 0.6.1, but installation seems correct nonetheless
```

Comment:
We should fix this in that package, otherwise we force all of its dependencies to skip this test.

Reply:
that is correct, @ocefpaf.
However, last version of `properties` (0.6.1) was 6 years ago, and I am no maintainer of it.

Error is:
![image](https://github.com/user-attachments/assets/43e9934f-6825-4a64-b90a-3af50c731792)

`properties` depends on `vectormath`. Last version was also six years ago.

I see the conda recipes for both `properties` and `vectormath` use the package from PyPI:
- https://github.com/conda-forge/properties-feedstock
- https://github.com/conda-forge/vectormath-feedstock

Could it be that `pip check` is expecting an information that was not yet available in pip packages 6 years ago?

@conda-forge/help-python, what would you recommend ?

Reply:
Maybe it is not worth messing with this. Let's merge this with the skip, but please open an issue with the pip-check fai l in those feedstock so we can track this.

Reply:
👍 
created two issues:
- https://github.com/conda-forge/properties-feedstock/issues/15
- https://github.com/conda-forge/vectormath-feedstock/issues/6